### PR TITLE
feat(#10): add settings to toggle 0, Home and End

### PR DESCRIPTION
**Pull request by tuurep**
Issue: #10

The checkboxes are located in Manage Extension -> Preferences

### CODE REVIEW

File: youtube-disable-number-seek.js [hunk 1]
```diff
@@ -6,6 +6,24 @@
...
+
+browser.storage.local.get(keys).then(result => {
+    for (const key of keys) {
+        if (result[key] !== undefined) {
+            settings[key] = result[key];
+        } else {
+            browser.storage.local.set({[key]: true});
+        }
+    }
+});
```
Comment (tuurep) [inline youtube-disable-number-seek.js:27]: Dealing with asynchronous functions is new to me, does anything look like a problem here?

**Comment by timmontague** [comment]
I won’t have time to review this for a few days. Wife is having a baby

**Comment by tuurep** [comment]
Ok, no hurry. And congratulations! :smile:

I still have to fix the first-time initialization of the storage and checkboxes as well.

**Comment by timmontague** [comment]
Looks good to me. Thanks for your help!

**Pull request merged**